test/powershell/Modules/Microsoft.PowerShell.Core/History.Tests.ps1

```diff
@@ -110,4 +110,13 @@ Describe "History cmdlet test cases" -Tags "CI" {
 
         $errorResult | Should -BeExactly 'CommandNotFoundException'
     }
+
+    It "HistoryInfo calculates Duration" {
+        $ctor = [Microsoft.PowerShell.Commands.HistoryInfo].GetConstructors([Reflection.BindingFlags]::NonPublic -bor [Reflection.BindingFlags]::Instance).Where{$_.GetParameters().Length -eq 5}
```

Comment:
Seems like this test should just get the first item from item from `Get-History` and validate that Duration = End - Start?

Reply:
@SteveL-MSFT  Am I guaranteed to have something in the history at that moment?

Reply:
Fixed